cylc/flow/parsec/fileparse.py

```diff
@@ -456,7 +462,7 @@ def read_and_proc(
     do_jinja2 = True
     do_contin = True
 
-    extra_vars = process_plugins(Path(fpath).parent, opts)
+    extra_vars = process_plugins(fpath, opts)
```

Comment:
Maybe add docstring for a description of the args?

Reply:
Do you mean a comment here?
`process_plugins` already has argument descriptions?

Reply:
I mean for this function. From this change, it isn't clear whether `fpath` is supposed to be a file or a directory anymore